### DISCUSSION TITLE
变量forward_to_master默认true代表转发还是不转发

### DISCUSSION THREAD

**Discussion by limo520** [apache/doris#12653] [A - General / Q&A] [answered]
https://doris.apache.org/docs/dev/advanced/variables.html#supported-variables

英文文档写的“ The default is true, which means no forwarding”
中文文档写的“默认为 true，即转发”

另外，文档英文和中文哪个会更新得更及时一些？

**Comment by JNSimba** [comment, marked as answer]
It is forwarding, Chinese and English are generally modified synchronously
